Mike's writeup

murphnomer opened the pull request.

murphnomer committed:
Mike's writeup

jaredscarr commented:
If you add "elixer" after the three ``` at the beginning of the code blocks you will get syntax highlighting.

murphnomer committed:
Update REPORT.md

Added language tag to code block for syntax highlighting
Added struct module to example program

jaredscarr reviewed (approved):
Nice struct!
